script/openqa-clone-job

```diff
@@ -41,8 +41,8 @@ settings can be modified.
   # clones job 42 (and any existing parents) from "openqa.opensuse.org" to the openQA instance "openqa.example.com"
   openqa-clone-job --skip-download --from https://openqa.opensuse.org --host openqa.example.com 42
 
-  # clones job 42 (and any existing parents) within the local openQA instance modifying some job settings
-  openqa-clone-job --within-instance 42 MAKETESTSNAPSHOTS=1 TEST+=:PR-123 FOOBAR=
+  # clones job 42 (and any existing parents) within "openqa.opensuse.org" modifying some job settings
+  openqa-clone-job --within-instance https://openqa.opensuse.org 42 MAKETESTSNAPSHOTS=1 TEST+=:PR-123 FOOBAR=
```

Comment:
https://github.com/os-autoinst/openQA/pull/5228/files#diff-11833f82f8f8b011f57f337feeb46afb346d9a9adbc60ce16d1e8704f5f36bd3R37 mentions a complete URL as *primary* example so I would work with that

```suggestion
  openqa-clone-job --within-instance https://openqa.opensuse.org/t42 MAKETESTSNAPSHOTS=1 TEST+=:PR-123 FOOBAR=
```